src/image-map.ts

```diff
@@ -0,0 +1,91 @@
+export const icons = {
+  home: "v1699887218/icons/gvqxjxcw07ei2kkmwnes.svg",
+  send: "v1699893673/icons/rmvytilpjgvh3pgwc8ph.svg",
+  rocket: "v1699894015/icons/u26xm5hteot875ozfnk9.svg",
+  edit: "v1699894149/icons/qxlunbg5tfrdherzsvlt.svg",
+  hamburger: "v1699895123/icons/jaqdnomtbhqvjcap962u.svg",
+  cross: "v1699895179/icons/xigimrbivcaxt4omxamp.svg",
+  bell: "v1699895207/icons/ecejtvqerx81prxyxh9b.svg",
+  twitter: "v1699895251/icons/bq6a50xtkvnzhltfggzz.svg",
+  worksheet: "v1699895300/icons/ez1s6mtpe5jkunnxzvlw.svg",
+  facebook: "v1699895330/icons/iojlvh3o5lshy2jupyph.svg",
+  share: "v1699895363/icons/agcmduftef3wcla6gzec.svg",
+  "arrow-right": "v1699895398/icons/fv0z57zerrioft52dd9n.svg",
+  "worksheet-3": "v1699895429/icons/bzhojpjxp9rukdvh7daz.svg",
+  "chevron-right": "v1699895475/icons/rhzft6wnpdp3nvsmlyei.svg",
+  save: "v1699895505/icons/rh1ahwwtbemvz0ihluew.svg",
+  "quiz-3": "v1699895534/icons/zoayhgtrotv32fad7d3k.svg",
+  "chevron-down": "v1699953557/icons/botfld6brychmttwtv6u.svg",
+  linkedin: "v1699953592/icons/leqneklorqqzb1zo6rf1.svg",
+  "magic-carpet": "v1699953622/icons/jifivg9xxm7sb0fjdilm.svg",
+  books: "v1699953657/icons/hz4l3iq6i68kazvkvorq.svg",
+  "supervision-level": "v1699953696/icons/cwqhknapp3maa4g0t3lj.svg",
+  "quiz-white": "v1699953730/icons/gpcehpgr9mqoumsa25xe.svg",
+  "arrow-left": "v1699953763/icons/ejrm9dpgpzb7ddbo8z2i.svg",
+  "additional-material": "v1699953798/icons/ntoq4tqvcm2uj1pajubt.svg",
+  "slide-deck-3": "v1699953830/icons/sm92moja9d5utu3cj16c.svg",
+  "sign-language": "v1699953861/icons/ns94ozvozzi22enxkx0x.svg",
+  external: "v1699953892/icons/hlxmejse3mcr4tqo6t8u.svg",
+  "equipment-required": "v1699953925/icons/pw22bdhj2vrzfv2ogi4e.svg",
+  "chevron-left": "v1699953962/icons/xpp10hijwzhyeqt6ylqf.svg",
+  download: "v1699953991/icons/dk0f6a6hdpzxftjosngn.svg",
+  search: "v1704901279/icons/canbi3fuz5fanzom2hvi.svg",
+  "chevron-up": "v1699954058/icons/pay71thmhhylj7z28sj1.svg",
+  go: "v1699954090/icons/vdzptyvmitylra8x4usy.svg",
+  copyright: "v1699954118/icons/boiod3rflocgsnfokyo8.svg",
+  "arrow-down": "v1699954152/icons/wpfmbmwpyfinipg0d61y.svg",
+  project: "v1699954186/icons/zofq5pheud6spnwjpewk.svg",
+  "slide-deck": "v1699954241/icons/sjjy5f3g4eciwcuxxr33.svg",
+  "content-guidance": "v1699954277/icons/tm3uhcqenaznq4fxys7j.svg",
+  tick: "v1699954310/icons/efd3esaor6zqk7seh6kt.svg",
+  instagram: "v1699954343/icons/ayfeljric0kkimdymvva.svg",
+  dot: "v1699954371/icons/knykdclphkm8lgff4u2g.svg",
+  "dot-png": "v1699954394/icons/qecbh291nzwmhcvqayqd.png",
+  warning: "v1704901279/icons/zzszodmk7fvxm9xzzg9s.svg",
+  "lightbulb-yellow": "v1705078631/icons/q2v4sqxouy1ngcajoavv.svg",
+  lightbulb: "v1705078631/icons/zldisxmbff36z68rwcef.svg",
+  quiz: "v1705416077/icons/kaaizjcudy0jfgfrrdel.svg",
+  video: "v1705416078/icons/wzey1zfxrvv3apeebbf5.svg",
+  intro: "v1705662092/icons/pl7bnmb13txese9yxkjv.svg",
+  "subject-art": "subject-icons/art.svg",
```

Comment:
Depending on the context in which the icon is used I would either recommend `Subject: Art`, `Art` or an empty string (in the case where the subject name is visually exposed as text)

Reply:
Perfect thanks 🙇🏼 .

The subject is also displayed below the icon so the empty string would seem to make sense to avoid repetition.

<img width="609" alt="Screenshot 2024-01-30 at 17 53 21" src="https://github.com/oaknational/oak-components/assets/122096/b91f9c38-d801-48f3-b7cf-83895e3603db">